packages/core/src/sfdxwrappers/CreateSourcePackageImpl.ts

```diff
@@ -156,9 +237,23 @@ export default class CreateSourcePackageImpl {
     };
   }
 
+  private  printClassesIdentified(fetchedClasses:FileDescriptor[]) {
 
- 
+  
+    if(fetchedClasses===null || fetchedClasses===undefined)
+        return;
+     
+    let table = new Table({
+      head: ["Class","Path", "Error"],
+    });
 
+    for (let fetchedClass of fetchedClasses) {
+      let item = [fetchedClass.name, fetchedClass.filepath,fetchedClass.error?fetchedClass.error:"N/A"];
```

Comment:
Duplicated fetchedClass.error ?

Reply:
Right it's a ternary operator, got confused without spacing

Reply:
terenary operator,  if null print N/A